fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -358,6 +360,7 @@ def __init__(
                 if not hasattr(process_group_reduce_scatter, "allgather") and hasattr(
                     process_group_reduce_scatter, "rank"
                 ):
+                    # FIXME: common test func for dummy PG.
```

Comment:
I think the comment on the next line means that this case is caused by the usage of a dummy PG in our unit test, i.e. this case is not a production use case.